### PR TITLE
Update domains-faq.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the subdomain

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/962

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/setup/domains-faq.md [hunk 1]
```diff
@@ -274,6 +274,10 @@ When you add a subdomain, it is automatically verified based on the parent domai
 When you add multiple domains to Office 365, you can host any of the services (like email) on any of the domains you've added.  *When you change your email to Office 365, by updating a domain's MX record, ALL email sent to that domain will start coming to Office 365.* 
  
 ::: moniker range="o365-worldwide"
+
+> [!NOTE]
+> If you have already added a domain contoso.com to a O365 tenant, we can still add the sub domain xyz.contoso.com to another Office365 tenant, when adding the subdomain you will be prompted to add a txt record in the dns hosting provider.
```
Comment (JohanFreelancer9) [inline Office365-Admin/setup/domains-faq.md]:
```suggestion
> If you have already added a contoso.com domain to an Office 365 tenant, you can still add the subdomain xyz.contoso.com to another Office 365 tenant. When adding the subdomain, you will be prompted to add a TXT record in the DNS hosting provider.
```

**Comment by mypil** [comment]
@pebaum - PR has been copy edited and is ready for final review. This is based on issue #962

Thank you.

cc: @kenwith

**Pull request merged**